lib/hexpm/repository/package.ex

```diff
@@ -115,6 +115,20 @@ defmodule Hexpm.Repository.Package do
     end
   end
 
+  defp search_param("depends", search, query) do
+    dependencies = from(rq in Requirement,
+      join: p in assoc(rq, :dependency),
+      join: rl in assoc(rq, :release),
+      select: rl.package_id,
+      distinct: rl.package_id,
+      where: p.name == ^search,
+    )
+
+    from(p in query,
+      join: d in subquery(dependencies),
```

Comment:
When I tried grouping them in a join Postgres threw me this error:

```
column "p0.id" must appear in the GROUP BY clause or be used in an aggregate function
```

Counting on `GROUP BY` doesn't sound correct while aggregate function seems to duplicate other cases. 🤔